roles/ansible-tdp-common-actions/tasks/deploy-ranger-hbase-policy.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: "Ranger: get policy list"
+  uri:
+    url: "{{ ranger_hdfs_install_properties.POLICY_MGR_URL }}/service/public/v2/api/policy"
+    headers:
+      Accept: application/json
+    url_username: "{{ ranger_admin_user }}"
+    url_password: "{{ ranger_admin_password }}"
+    force_basic_auth: yes
+    return_content: yes
+  register: jsoncontent
+
+- name: Get policy ids
+  vars:
+    hbase_policy_query: "[? name == '{{item.name}}'].id"
+  set_fact:
+    hbase_policy_id: "{{ jsoncontent.json | community.general.json_query(hbase_policy_query) }}"
+
+- name: Create hbase policy for tdp_user
```

Comment:
Is this meant to be reusable ? If I recall, this post request replaces the policy at the `id` specified, therefore if there are other users and you only put tdp_user each time, you'll lose the others.

Reply:
Might be okay for now, we might want to create an issue not to lose track of this.

Reply:
I'm not sure I completely understood the users part. 

From what I got the post will check if the name with that policy exists. 
- If it does exist, hbase-create will ignore it since it doesn't pass the when condition. 
- If it doesn't exist it will create a new one in the next free id. 
- It won't edit a policy at a specified ID. 

About the users, what I could see is the following:
- If a policy is created with specific users (1 or multiple), this policy can't be edited after it's creation by recalling this current tasks. If it's reused with the same name, the policy won't be edited, it will just ignore it since the id already exists.
- If a new policy with different name is created and this tasks are re-used to do so, a new policy is created and the other's aren't deleted.

I'm not entirely sure if this covers what you were talking about?

I'll try to add a way of adding a task to edit specific policies too (like adding users, etc).

Reply:
Talked with @DanielJohnHarty. I'll correct the name as it is a bit more general than for simply tdp_user.  
I'll do a more general policy deployment later that merges the other ones (hdfs, etc), as tasks are repeated across deployments.